app/jsx/pages/ItemPage.jsx

```diff
@@ -11,15 +11,45 @@ import TabsComp from '../components/TabsComp.jsx'
 import JumpComp from '../components/JumpComp.jsx'
 import FooterComp from '../components/FooterComp.jsx'
 
+const anchors = ['main', 'supplemental', 'metrics', 'author',
+             'article_abstract', 'article_main', 'article_references']
+
 class ItemPage extends PageBase {
-  state = { currentTab: 1 }
+  static propTypes = {
+    currentTab: React.PropTypes.oneOf(anchors)
+  }
 
   // PageBase will fetch the following URL for us, and place the results in this.state.pageData
   pageDataURL() {
     return "/api/item/" + this.props.params.itemID
   }
 
-  changeTab = tab_id => { this.setState({currentTab: tab_id}) }
+  tabNameFromHash() {
+    return location.hash.toLowerCase().replace(/^#/, "")
+  }
+
+  // currentTab should be 'main' whenever hash starts with 'article_" (or is empty)
+  articleHashHandler = h => {
+    return ((h.startsWith("article") || h=='') ? "main" : h)
+  }
```

Comment:
For future ref, I think this can be simplified to a one-liner:
`articleHashHandler = h => ((h.startsWith("article") || h=='') ? "main" : h)`